gravatar-quickeditor/src/main/res/values/strings.xml

```diff
@@ -28,6 +28,8 @@
     <string name="avatar_picker_network_error_message">Uh oh! Looks like you\'re not connected to the internet.</string>
     <string name="login_required">Login required</string>
     <string name="login_required_message">To modify your Gravatar profile you need to log&#160;in first</string>
+    <string name="oauth_wrong_email_authenticated_error_title">Ooops</string>
```

Comment:
We already have that one, I wonder if we should reuse it?

Reply:
Yeah. I miss that one. I saw the others with more text, but not that one. 

Updated.